docs/webiny-apps/page-builder/theme-development/stylesheets.md

```diff
@@ -6,8 +6,8 @@ sidebar_label: Stylesheets
 
 If you've gone through the default theme configuration file you've probably noticed that it's mostly just a list of class names and few references to React components. We didn't talk about any stylesheets so far. Working with stylesheets in Webiny is pretty straight forward.
```

Comment:
Please update “few” to “a few”, and “straightforward” at the end of the sentence.

Reply:
I noticed a few other issues in wording so I pushed some changes for clarity.